assets/scss/pages/_home.scss

```diff
@@ -7,5 +7,134 @@ body#top.home {
+            font-family: "interstate", Sans-serif, serif;
+            font-weight: bold;
+            text-shadow: .1rem .1rem .1rem $black-color;
+        }
+    }
+
+    .lg-mission-statement {
+        h2 {
+            font-size: 2.4rem;
+            margin-bottom: 1.5rem;
+        }
+    }
+
+    .lg-state-of-the-art-equipment-section {
+        .entry-content-wrapper {
+            @include media("<tablet") {
+                display: flex;
+                flex-direction: column-reverse;
+            }
+        }
+    }
+
+    .lg-call-to-action-section {
+        margin-bottom: 1.5rem;
+
+        h2 {
+            color: $white-color;
+            font-size: 2.6rem;
+            font-weight: bold;
+            text-align: center;
+            line-height: 3.5rem;
+        }
+    }
+
+    .lg-contact-section {
+        background-color: $dark-blue-color;
+        margin-bottom: 3.5rem;
+
+        h2 {
+            color: $white-color;
+        }
+
+        .flex_column_table {
+            @include media("<=932px") {
+                display: flex;
+                flex-direction: column;
+            }
+
+            .lg-contact-form-section {
+                p,
+                .gfield_label {
+                    color: $white-color;
+                }
+
+                input[type="text"] {
+                    border-radius: .4rem;
+                }
+
+                input[type="submit"] {
+                    background-color: $red-color;
+                    border-radius: .4rem;
+                    font-size: 2rem;
+
+                    &:hover {
+                        background-color: $white-color;
+                        color: $red-color;
+                    }
+                }
+
+                textarea {
+                    border-radius: .4rem;
+                }
+
+                .gform_confirmation_message_1 {
+                    color: $white-color;
+                    font-size: 1.8rem;
+                }
+
+                @include media("<932px") {
+                    width: 100%;
+                }
+            }
+
+            .av-flex-placeholder {
+                width: 7.5%;
+            }
+
+            .lg-address-section {
+                @include media(">932px") {
+                    border-left: .2rem solid $white-color;
+                    margin-left: 0;
+                    padding-left: 11rem;
+                }
+
+                @include media("<932px") {
+                    border-top: .2rem solid $white-color;
+                    padding-top: 3.5rem;
+                    width: 100%;
+                    margin-left: 0;
+                }
+
+                h3,
+                p,
+                a,
+                strong {
+                    color: $white-color;
+                    font-family: "interstate", serif;
+                }
+
+                p,
+                a {
+                    font-weight: 400;
+                }
+            }
+        }
+    }
```

Comment:
I would extract this chunk out into a partial, maybe in a /partials/home-partials directory. The styling for this one section is larger than the whole rest of the file, so extracting it out will make both files easier to work with.